COMPILING.txt

```diff
@@ -58,47 +51,28 @@ irc.freenode.net.
     Debian/Ubuntu
     -------------
     g++
-    automake
-    pkg-config
-    libcurl-dev
-    libsigc++-dev
+    cmake
+    mesa-common-dev
+    libfreeimage-dev
+    libglew-dev
+    libsigc++-2.0
     libsigc++-2.0-dev
-    libsdl2-dev
-    libsdl2-image-dev
-    libfreetype6-dev
     libvorbis-dev
-    libpng-dev
     libassimp-dev >= 3.2
-    mesa-common-dev
-
-   If you are building with CMake instead of Autotools, you'll need to install
-   the cmake package instead of the automake package.
+    libsdl2-dev
+    libsdl2-image-dev
 
    If your platform doesn't have assimp 3.2, you'll need to build it from
    source. It's available in pioneer-thirdparty, see below.
 
-1.1.a Linux - Autotools
------------------------
-
-1. Run ./bootstrap to generate your 'configure' file
-
-2. Run ./configure to configure the build. If you're using the
-   pioneer-thirdparty repository, pass
-   --with-thirdparty=/path/to/pioneer-thirdparty to configure.
-
-3. Run make to build everything
-
-1.1.b Linux - CMake
--------------------
-
-1. Run './bootstrap cmake' to generate CMake build files in the 'build' folder.
+2. Run './bootstrap cmake' to generate CMake build files in the 'build' folder.
```

Comment:
just checking: does bootstrap still need the `cmake` argument?

Reply:
It's supported but not required - I'll remove the mention of it here.